### DISCUSSION TITLE
No alerts in SOC Web Interface (Surcata or Zeek)

### DISCUSSION THREAD

**Discussion by Kilt-Guy** [Security-Onion-Solutions/securityonion#11371] [2.4]
### Version

2.4.1

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

16 GB

### Storage for /

152

### Storage for /nsm

303

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

Standalone installation on a NUC, plenty of horsepower for it, but keeps getting Elastic errors when loading (more below), but so-status shows everything green.  We've reloaded several times with (seemingly) the same error messages.  Zeek and Suricata are generating alerts in the respective alert files found under /nsm/ so it's not the monitoring interface, nor is it a misconfiguration of rules for the NIDS, it's all surrounding the ingestion of those alerts for the SOC web interface.

The sosetup.log had errors, which are related to the state.highstate errors recorded below.  Additionally, there were a slew (7672) of fatal alerts for "bad_certificate" in the logstash.log for the javax.net.ssl.SSLHandshakeException.  

No other significant errors in any of the other log files

There were two errors from state.highstate:
ID: so-elastic-fleet-auto-configure-logstash-outputs
Function: cmd.run
Name: /usr/sbin/so-elastic-fleet-outputs-update
Result: False
Comment: Attempt 1: Returned a result of "False", with the following comment: "Command "/usr/sbin/so-elastic-fleet-outputs-update" run"
 Command "/usr/sbin/so-elastic-fleet-outputs-update" run
Started: 20:45:46.671782
Duration: 30092.214 ms
Changes:
blah, blah, blah
stdout: Failed to query for Logstash Outputs
------------
ID: so-elastic-fleet-auto-configure-elasticsearch-urls
Function: cmd.run
Name: /usr/sbin/so-elastic-fleet-es-url-update
Result: False
Comment: Attempt 1: Returned a result of "False", with the following comment: "Command "/usr/sbin/so-elastic-fleet-es0url-update" run"
  Command "usr/sbin/so-elastic-fleet-es-url-update" run
Started: 20:46:19.375400
Duration: 30084.575 ms
Changes: 
blah, blah, blah
stdout: Failed to query for current Fleet Server Elasticsearch URLs...

=-=-=-=-=-=-=-=-=

Now, to the crux of the problem.  Where in the pipeline is the issue?  My bet is Logstash, considering that's where I was seeing errors, but I don't know how to resolve a bad_certificate error for the one system when everything else seems to work just fine.  Especially considering this has happened through multiple reloads/rebuilds of the system.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment]
The main issue is Elastic is more than likely being OOM killed. Is it possible to bump the RAM up?

https://docs.securityonion.net/en/2.4/hardware.html#standalone-deployments

**Reply by asmaaelkandoussi** [reply]
hi the same problem with security onion 2.4 
Any update to get alert !

**Reply by dougburks** [reply]
@asmaaelkandoussi From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:
_Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and in that new discussion you can provide a hyperlink to the related discussion._

I've responded to your new discussion at https://github.com/Security-Onion-Solutions/securityonion/discussions/11446.
